src/MenuItems.js

```diff
@@ -31,7 +31,7 @@ class MenuItems extends React.Component {
 
```

Comment:
If you can remove some extra blank lines, it will look better.

Reply:
Will do!